workshop/Lab3/README.md

```diff
@@ -1,45 +1,47 @@
-# Lab 3: Scale and update apps natively, building multi-tier applications.
+# Lab 3: Scale and update apps natively by building multi-tier applications
```

Comment:
I think the comma is more correct here since we're doing 2 things, not 1.

Reply:
Ah okay. Done.